cmd/zdb/zdb.c

```diff
@@ -111,7 +111,7 @@ typedef void object_viewer_t(objset_t *, uint64_t, void *data, size_t size);
 
 uint64_t *zopt_object = NULL;
 static unsigned zopt_objects = 0;
-uint64_t max_inflight = 1000;
+uint64_t max_inflight_bytes = 16 * 1024 * 1024; /* 16MB */
```

Comment:
One max size block seems a bit stingy. the old default was 80MB (with 8K blocks). How about something like 256MB?

Reply:
I had thought the default was 8MB with 8K block and also noticed that we cap the arc at 256MB. So I sized this using the same default of 1/16th of the ARC and that's why I set it to 16MB. Do you think we should raise the ARC size for zdb too?

Reply:
Fixed